dev-docs/source/BuildingWithCMake.rst

```diff
@@ -64,7 +75,7 @@ From the command line
 * If you want to use a specific generator (run ``cmake --help`` for a list of available generators for your platform), use the ``-G`` option, e.g. ``cmake -G"NMake Makefiles" /path/to/Mantid``.
 * If you want to set the build type (e.g. Release, Debug) you can run cmake with the ``-i`` option or by passing the argument ``-DCMAKE_BUILD_TYPE=Debug`` to cmake. The default is Release.
```

Comment:
I believe this is a good place to advertise this build type: `-DCMAKE_BUILD_TYPE=RelWithDebInfo`